website/client/src/components/achievements/newStuff.vue

```diff
@@ -4,21 +4,36 @@
     size="lg"
     :hide-header="true"
     :hide-footer="true"
-    no-close-on-esc="no-close-on-esc"
-    no-close-on-backdrop="no-close-on-backdrop"
+    no-close-on-esc
+    no-close-on-backdrop
   >
     <div class="modal-body">
       <div
-        class="static-view"
-        v-html="html"
-      ></div>
+        v-for="(post, index) in posts"
+        :key="index"
+        :class="index == (posts.length - 1) ? 'bailey bailey-last' : 'bailey'"
+      >
+        <small
+          v-if="!post.published"
+          class="draft"
+        >DRAFT</small><h2 class="title">
+          {{ post.publishDate | date }} - {{ post.title }}
```

Comment:
I don't know if you're interested in feedback about this sort of thing (so feel free to ignore this if it's not appropriate!) but is it possible for `post.publishDate` to use the user's preferred date format? It's the sort of thing that they're likely to ask for. Here's an example of how the format is used in case it speeds things up:
https://github.com/HabitRPG/habitica/blob/be622e8b10ce7f9e59056c77ad01762320d85064/website/client/src/components/userMenu/profile.vue#L775-L778

Reply:
That's a good point! I'll make that change. Thanks.